### PR TITLE
Stripe init

### PR DESCRIPTION
**Pull request by z-rodney**
Started integrating stripe to accept payment, and redirect to order details on successful payment

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,5 @@
 node_modules
-server/public/bundle.js
-server/public/bundle.js.map
-server/public/bundle.js.LICENSE.txt
+server/public/**.js
```
Comment (adam-adair) [inline .gitignore]: This doesn't seem to work. Looks like you've included bundle.js.map in this PR.

File: client/components/Cart.js [hunk 2]
```diff
@@ -63,48 +66,39 @@ export class Cart extends Component {
...
-            )
-          })}
+                  </h5>
+                  <button onClick={()=>this.removeItem(cartItem)}>Remove from cart</button>
+                  <hr />
+                </div>
+              )
+            })}
+        </div>
+        <Checkout cart={ cart } artworks={ artworks } />
```
Comment (adam-adair) [inline client/components/Cart.js]: I think we should have some logic here that prevents checkout if you have an invalid qty in your cart.

File: server/checkout/index.js [hunk 3]
```diff
@@ -0,0 +1,226 @@
+/* eslint-disable camelcase */
```
Comment (adam-adair) [inline server/checkout/index.js:1]: I don't know anything about what's going on here, but it does seem to work on my end when running with the stripe CLI listener. So, great!

**Review by adam-adair — COMMENTED** [review]
I've left a few relatively minor comments on this. I think it would be helpful if you merged dev into this so that a) admins could see orders once an order is completed and make sure everything looks right in that view, and b) some merge conflicts would be resolved, I think in /api/order

File: .gitignore (hunk 1 above)
Comment (z-rodney) [reply .gitignore]: fixed

**Review by zoegreene — APPROVED** [review]
ready freddy

**Pull request merged**